pipeline_products_session/jwst-data-products-part1.ipynb

```diff
@@ -0,0 +1,961 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fits.info(uncal_obs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "4.<font color='white'>-</font>Examining an exposure with astropy<a class=\"anchor\" id=\"astro\"></a>\n",
+    "------------------\n",
+    "\n",
+    "Many of you may be familiar with using [astropy](https://docs.astropy.org/en/stable/) to examine data. Here, we will take a look at the format and headers using standard ```astropy``` tools. \n",
+    "\n",
+    "### A.<font color='white'>-</font>Format<a class=\"anchor\" id=\"astro-format\"></a>\n",
+    "\n",
+    "Below, we see the typical extensions in a raw JWST data file. All data related to the product are contained in one or more FITS IMAGE or BINTABLE extensions, and the header of each extension may contain keywords that are uniquely related to that extension.\n",
+    "\n",
+    "* PRIMARY: The primary Header Data Unit (HDU) only contains header information, in the form of keyword records, with an empty data array (indicated by the occurence of NAXIS=0 in the primary header. Meta data that pertains to the entire product is stored in keywords in the primary header. Meta data related to specific extensions (see below) is stored in keywords in the headers of each extension.\n",
+    "* SCI: 4-D data array containing the raw pixel values. The first two dimensions are equal to the size of the detector readout, with the data from multiple groups (NGROUPS) within each integration stored along the 3rd axis, and the multiple integrations (NINTS) stored along the 4th axis.\n",
+    "* ZEROFRAME: 3-D data array containing the pixel values of the zero-frame for each integration in the exposure, where each plane of the cube corresponds to a given integration. Only appears if the zero-frame data were requested to be downlinked separately.\n",
+    "* GROUP: A table of meta data for some (or all) of the data groups.\n",
+    "* INT_TIMES: A table of begining, middle, and end time stamps for each integration in the exposure.\n",
+    "* ADSF: The data model meta data.\n",
+    "\n",
+    "Additional extensions can be included for certain instruments and readout types. The [JWST software readthedocs](https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/science_products.html) contains the most up-to-date information about JWST formats. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fits.info(uncal_obs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can grab the data to examine it: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "science_data = fits.getdata(uncal_obs,1)\n",
```

Comment:
Maybe switch the 1 to 'SCI' here in order to be consistent with the 'with' statement below, and just to be more explicit?

Reply:
Didn't even know that was possible, I fixed it